Solutions/Darktrace/Analytic Rules/CreateAlertFromSystemStatus.yaml

```diff
@@ -14,20 +14,44 @@ triggerThreshold: 0
 tactics: # none
 relevantTechniques: # none
 query: |
-  darktrace_model_alerts_CL
+  darktrace_model_alerts_CL //anything starting with 'Dt' is not an ASIM mapping 
   | where dtProduct_s =="System Alert"
-  | extend EventVendor = "Darktrace"
-  | extend EventProduct = "Enterprise Immune System"
+  | extend EventVendor="Darktrace", EventProduct="Darktrace DETECT"
+  | project-rename ThreatCategory=dtProduct_s, EventStartTime=time_s, NetworkRuleName=friendlyName_s, SrcIpAddr=deviceIP_s, SrcHostname=hostname_s, ThreatRiskLevel=priority_code_d, ThreatRiskCategory=priority_s, DtSeverity=Severity, DtName=name_s, DtStatus=status_s, DtMessage=Message, DtURL=url_s, DtUUID=uuid_g
 eventGroupingSettings:
   aggregationKind: AlertPerResult
-entityMappings: # none
+entityMappings:
+  - entityType: Host
+    fieldMappings:
+      - identifier: HostName
+        columnName: SrcHostname
+  - entityType: IP
+    fieldMappings:
+      - identifier: Address
+        columnName: SrcIpAddr
 customDetails:
-  Title: friendlyName_s
-  EventMessage: Message
+  EventStartTime: EventStartTime
+  NetworkRuleName: NetworkRuleName
+  ThreatRiskLevel: ThreatRiskLevel
+  ThreatRiskCategory: ThreatRiskCategory
+  DtName: DtName
+  DtStatus: DtStatus
+  DtMessage: DtMessage
 alertDetailsOverride:
-  alertDisplayNameFormat: 'Darktrace: {{friendlyName_s}}'
-  alertDescriptionFormat: "{{Message}}"
+  alertDisplayNameFormat: 'Darktrace: {{ThreatRiskLevel}} - {{NetworkRuleName}}'
+  alertDescriptionFormat: '{{DtMessage}}'
   alertTacticsColumnName: # none
   alertSeverityColumnName: # none 
-version: 1.0.0
-kind: NRT
+  alertDynamicProperties:
+    - alertProperty: AlertLink
+      value: DtURL
+    - alertProperty: ProviderName
+      value: EventVendor
+    - alertProperty: ProductName
+      value: EventProduct
+    - alertProperty: ProductComponentName
+      value: ThreatCategory
+    - alertProperty: Severity
```

Comment:
Why are we using different techniques here?
One query maps Severity using alertSeverityColumnName, other using alertProperty.
This is the best and latest and thus should be adopted.

Reply:
@seanmacdonald8 You are right, the style document requires improvement and we will take care of it very soon. Thanks for pointing it out. For CreateAlertFromSystemStatus.yaml, can we map values from 1 - 10 to know severities (High, Medium, Low, Informational)? If yes, that would be better for the query.

Reply:
@devikamehra we are happy for system status alerts to have a flat severity - these are just meta-alerts about system health, they are not security alerts per se. If you're happy with that please proceed with the merge.

Reply:
@devikamehra I agree with Justas as we were hoping to just leave the system status alerts at the "informational" level. Thanks!